bindings/experimental/distrdf/python/DistRDF/Ranges.py

```diff
@@ -321,82 +324,95 @@ def get_clustered_ranges(clustersinfiles, npartitions, treename, friend_info):
-    2. ``end``: The maximum entry among all the clusters considered in a
-        given partition. The offset of the first cluster of the list is
-        subtracted. This is useful to keep the reference of the range with
-        respect to the current files (see below).
-    3. ``filelist``: The list of files that are span between entries
-        ``start`` and ``end``::
-            Filelist: [file_1,file_2,file_3,file_4]
-            Clustered range: [0,150]
-            file_1 holds entries [0, 100]
-            file_2 holds entries [101, 200]
-            Then the clustered range should open [file_1, file_2]
-            Clustered range: [150,350]
-            file_3 holds entris [201, 300]
-            file_4 holds entries [301, 400]
-            Then the clustered range should open [file_2, file_3, file_4]
-        Each ``cluster`` namedtuple has a ``fileandindex`` namedtuple. The
-        second element of this tuple corresponds to the index of the file in
-        the input `TChain`. This way all files can be uniquely identified,
-        even if there is some repetition (e.g. when building a TChain with
-        multiple instances of the same file). The algorithm to retrieve the
-        correct files for each range takes the unique filenames from the list
-        of clusters and sorts them by their index to keep the original order.
-        In each file only the clusters needed to process the clustered range
-        will be read.
-    4. ``friend_info``: Information about friend trees.
-    In each range, the offset of the first file is always subtracted to the
-    ``start`` and ``end`` entries. This is needed to maintain a reference of
-    the entries of the range with respect to the list of files that hold
-    them. For example, given the following files::
-        tree10000entries10clusters.root --> 10000 entries, 10 clusters
-        tree20000entries10clusters.root --> 20000 entries, 10 clusters
-        tree30000entries10clusters.root --> 30000 entries, 10 clusters
-    Building 2 ranges will lead to the following tuples::
-        TreeRange(start=0,
-                end=20000,
-                filelist=['tree10000entries10clusters.root',
-                        'tree20000entries10clusters.root'],
-                friend_info=None)
-        TreeRange(start=10000,
-                end=50000,
-                filelist=['tree20000entries10clusters.root',
-                        'tree30000entries10clusters.root'],
-                friend_info=None)
-    The first ``TreeRange`` will read the first 10000 entries from the first
-    file, then switch to the second file and read the first 10000 entries.
-    The second ``TreeRange`` will start from entry number 10000 of the second
-    file up until the end of that file (entry number 20000), then switch to
-    the third file and read the whole 30000 entries there.
-    """
+    clustersbypartition = _n_even_chunks(clustersinfiles, npartitions)
```

Comment:
Could we make the naming consistent? E.g. `clustersbypartition` but `friend_info`

Reply:
If the name gets too long with `_`s, perhaps abbreviate some of its parts?

Reply:
I think I like without `_` better? Maybe the underscores improve readability, not sure. But in this function `friend_info` is the only one with underscore so it's easier to remove it there than adding it everywhere else.